code/datums/dna.dm

```diff
@@ -613,7 +613,7 @@ GLOBAL_LIST_INIT(total_uf_len_by_block, populate_total_uf_len_by_block())
 	if(dna.features["pod_hair"])
 		dna.features["pod_hair"] = GLOB.pod_hair_list[deconstruct_block(get_uni_feature_block(features, DNA_POD_HAIR_BLOCK), GLOB.pod_hair_list.len)]
 
-	for(var/obj/item/organ/external/external_organ as anything in external_organs)
+	for(var/obj/item/organ/external/external_organ in internal_organs)
```

Comment:
bro :sob:

Reply:
External organs are now in internal organs yes, it’s a lot better than having double definitions everywhere. Maybe I’ll have to rename it at one point